php/WP_Mock/Hook.php

```diff
@@ -22,6 +22,9 @@ abstract class Hook
     /** @var array<mixed> collection of processors */
     protected $processors = [];
 
+    /** @var array<mixed> collection of objects mapped to their Type hashes */
+    public static array $objects = [];
```

Comment:
```suggestion
    /** @var array<string> collection of objects mapped to their Type hashes */
    public static array $objects = [];
```

It is expected this list to hold string values